src/cplscheme/BaseCouplingScheme.cpp

```diff
@@ -409,7 +419,7 @@ bool BaseCouplingScheme::addComputedTime(
                 timeToAdd, _timeWindowSize - _computedTimeWindowPart + timeToAdd);
 
   if (hasTimeWindowSize()) {
```

Comment:
```suggestion
  //@todo: if-else branch is to some degree a duplicate of BaseCouplingScheme::reachedEndOfTimeWindow. Try to remove duplicate code.
  if (hasTimeWindowSize()) {
```
Let's take care of this after directly on develop and not in this PR, because it's not related. But please add the todo in the meantime that we don't forget it.